rebac-admin-backend/v1/identities.go

```diff
@@ -22,6 +22,8 @@ func (h handler) GetIdentities(w http.ResponseWriter, req *http.Request, params
 	}
 
 	response := resources.GetIdentitiesResponse{
+		Links:  resources.NewResponseLinks[resources.Identity](req.URL, identities),
```

Comment:
I think the `NewResponseLinks` method shouldn't be available for product services since it's actually an internal thing. But we cannot just do it by changing it's name to `newResponseLink`, because then it'll be also hidden from us here.

One solution is to define an alias for `PaginatedResponse` here (i.e., in the handlers package), and add the methods (e.g., `newResponseLink`) to the alias. This way the methods are only accessible to us and not to the product service.